### ISSUE TITLE
Acolyte Not Heal on Master and Party

### ISSUE THREAD

**Issue by oENUNAo** [OpenKore/openkore#1297]
This is my config

partySkill Heal {
	lvl 10
	dist 3
	maxCastTime 5
	minCastTime 2
	hp
	sp > 10
	homunculus_hp
	homunculus_sp
	homunculus_dead
	onAction
	whenStatusActive
	whenStatusInactive
	whenFollowing
	spirit
	amuletType
	aggressives
	monsters
	notMonsters
	monstersCount
	stopWhenHit 0
	inLockOnly 0
	notWhileSitting 0
	notInTown 0
	timeout 0
	disabled 0
	manualAI 0
	target xxxxxxxx
	target_hp < 80
	target_isJob
	target_isNotJob
	target_whenStatusActive
	target_whenStatusInactive
	target_aggressives
	target_monsters
	target_timeout 0
	target_deltaHp
	target_dead 0
	inInventory
	isSelfSkill 0
}

or use Heal but long time before kore use heal on master or party

and show this when lost master
I lost my master
Trying to find lost master
BUG: Actor::move(undef, undef) called!

Thank you...

**Comment by iRObotzz** [comment]
Not that anyone on this forum will report you, but you can leave out your bots real name when posting that config, do something like "target XXXXXXX" so that we understand that you are using that command line. For your own sake :P

Cant you leave the dist variable empty? so the dist line with read just "dist", maybe try that.

I just leave all my maxCastTime and minCastTime at 0 and my follow bots worked fine so far.

**Comment by bunnyy00** [comment]
dist 3 --> I just cancel this and all works well

**Comment by fadreus** [comment]
> dist 3
> maxCastTime 5
> minCastTime 2

LOL? Since when Heal got cast time. Even High Heal don't have that much cast time.
Why you set distance? Can heal far is better, why trouble yourself only heal when your slave in at your crotch?

> target_hp < 80

Let say your master have 2,000 HP, this setting only heal your party member when reach 79/2000.
(Of course it won't heal LOL). You should set `< 80%`. Heal at 80 is equivalent to heal < 2%. Don't be rambo.

Read the manual at http://openkore.com/index.php/Main_Page.

Just use like this
```
partySkill 28 {
sp >= 40
target_hp <= 80%
}
```

> I lost my master
> Trying to find lost master
> BUG: Actor::move(undef, undef) called!

Set your slave `route_step 19` & your master `route_step 9`.
All your 3 issues can be answered if you just do some research.

Read the manual 1st.
If that won't help, visit forum.
If that won't help, then check if similar issue existed (closed and open issue) before starting new issue.

**Comment by alisonrag** [comment]
enable bus in sys.txt and use this plugins:
Slave:
https://github.com/OpenKore/openkore/tree/master/plugins/needs-review/busFollow/trunk

Master:
https://github.com/OpenKore/openkore/tree/master/plugins/needs-review/wait4party/trunk

Both:
https://github.com/OpenKore/openkore/tree/master/plugins/needs-review/busParty/trunk

plus try the steps that our friends said to you before

**Comment by Junezavr** [comment]
@alisonrag error busParty
![1](https://user-images.githubusercontent.com/22251521/31934358-15ab4284-b8d6-11e7-81c8-ddedb0ab71f7.jpg)

**Comment by shace1093** [comment]
HI can anyone give me a free bot I dont know how to install it -_-

**Comment by fadreus** [comment]
@shace1093 Read at http://openkore.com how to install it.
